showcase.json

```diff
@@ -542,7 +542,7 @@
   {
     "id": "juno",
     "name": "Juno",
-    "oneLiner": "Build Web3 dapps like Web2",
+    "oneLiner": "Build Web3 dApps like Web2",
```

Comment:
was there another PR recently where all instances of `dApp` were normalized to `dapp`?

Reply:
There was, but David has requested that this single instance be reverted back to `dApp` to reflect the Juno branding properly.